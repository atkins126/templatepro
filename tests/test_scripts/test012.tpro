HTML Encoded
{{:myhtml}}

HTML Raw
{{:myhtml$}}