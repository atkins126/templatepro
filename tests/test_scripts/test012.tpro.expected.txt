HTML Encoded
&lt;div&gt;this &lt;strong&gt;HTML&lt;/strong&gt;&#345;&#353;&#269;&lt;/div&gt;

HTML Raw
<div>this <strong>HTML</strong>řšč</div>